chain-evm/Cargo.toml

```diff
@@ -4,10 +4,28 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-evm = "0.30.0"
+evm = "0.33.0"
 imhamt = { path = "../imhamt" }
 primitive-types = "0.10.1"
+base64 = { version = "0.13.0", default-features = false, features = ["alloc"] }
+blake2 = { git = "https://github.com/near/near-blake2.git", rev = "736ff607cc8160af87ffa697c14ebef85050138f", default-features = false }
+borsh = { version = "0.8.2", default-features = false }
+bn = { package = "aurora-bn", git = "https://github.com/aurora-is-near/aurora-bn.git", default-features = false, features = ["std"] }
```

Comment:
I would try to avoid this if possible or at least:
* Bind to a version
* Bind to a commit